src/main/java/uk/gov/hmcts/reform/pip/rules/repository/InMemoryRepository.java

```diff
@@ -35,8 +70,8 @@
             Hearing hearing = new Hearing();
             hearing.setHearingId(1);
             hearing.setCourtId(1);
-            hearing.setDate(LocalDate.now());
-            hearing.setTime(LocalTime.now());
+//            hearing.setDate(LocalDate.now());
```

Comment:
do we need any of this? youve added a whole repository, seems like we shouldnt need this builder that just created 1 with test data when youve spent all that time adding in the json files for all of them?

Reply:
I have removed it